common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -217,6 +219,8 @@ def create_response_element(self, **kwargs):
         """
 
         answer = kwargs.get('answer', None)
+        correcthint = kwargs.get('correcthint', '')
```

Comment:
Is this something new for this factory ? Why do we need this, add it to the doc string if we need this at all.